src/styles/antd-overrides/input.scss

```diff
@@ -149,6 +149,10 @@ input:-webkit-autofill-selected {
   color: var(--text-primary);
 }
 
+.ant-input-textarea {
+  width: 100%;
```

Comment:
**question** is this needed? Could we apply an inline style instead?

Reply:
Yeah inline style doesn't work